### ISSUE TITLE
live-featured-project doesn't always hide the project on 404

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6086]
### Description

You can set unshared projects as featured projects, so live-featured-project will not work. In that case, it should hide the embed - it sometimes does that, but often not.

### Steps to reproduce

1. Go to any profile with an unshared featured project (for example, https://scratch.mit.edu/users/mybearworld/)
2. Reload the page until you see Giga with a magnifying glass

### Demonstration(s) and additional context

![A demonstration](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/8a50d071-78a4-413a-a701-6b2b914e7105)

### Environment

Browser: Microsoft Edge 115.0.1851.0 (Official build) dev (64-bit) 
Scratch Addons version: 1.32.1
Operating system: Windows 11

**Comment by GrahamSH-LLK** [comment]
I can reproduce this by closing a tab and reopening it with cmd+shift+t. However I cannot get it working using your repro instructions. I tried ~50 times and didn't run into it, but that could just be me.

**Comment by mybearworld** [comment]
It mostly doesn't even need a reload for me:

https://github.com/ScratchAddons/ScratchAddons/assets/130385691/2aa2963f-cdf6-43f6-8096-a9e8fc380279

**Comment by iqnite** [comment]
It not intentional to hide the project if it can't be loaded, but it's a good idea!

**Comment by mybearworld** [comment]
Wait, really? That does sometimes happen. Is there a different bug here?

**Comment by iqnite** [comment]
Apparently, yes.

**Comment by mybearworld** [comment]
No, it is intentional, along with code for an alternative player when the project is unshared, which should probably be removed.
https://github.com/ScratchAddons/ScratchAddons/blob/0d66e2282337dfac90b5404b4fca2a4da0091f4f/addons/live-featured-project/script.js#L98-L103

**Comment by Chiroyce1** [comment]
> I can reproduce this by closing a tab and reopening it with cmd+shift+t. However I cannot get it working using your repro instructions. I tried ~50 times and didn't run into it, but that could just be me.

Is that on Chrome? on Mac with Firefox, even with cmd+shift+t, it flashes `Whoops! Our server` for a very small fraction of a second and then goes back to the thumbnail, which I think is the intended behaviour.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/97374054/4adb8be9-cbdc-4746-b0d6-b60a853a3bc1)

https://github.com/ScratchAddons/ScratchAddons/assets/97374054/dc9786af-f967-45a7-83dc-0df410317ed3
